source/community/activities/summer-of-code.html.md

```diff
@@ -42,6 +42,21 @@ When students approach you about the idea you listed:
 *   If you are redirecting students from your idea, please add [No longer taking applicants] to its title in the list below.
 *   Don't hesitate to reach out to the oVirt GSoC admins if you need help redirecting students.
 
+## oVirt Ideas for Google Summer of Code 2018
+
+### **Idea** Kubevirt/qemu-guest-agent
+
+**Description** The qemu-guest-agent is a useful tool to send system call to the
+guest operating system and KubeVirt project will benefit from it the same as
```

Comment:
> to send system call to the guest operating system

"to communicate directly with the guest operating system" sounds better to me

Reply:
Good catch, thanks.